src/ray/common/client_connection.h

```diff
@@ -48,7 +48,11 @@ Status ConnectSocketRetry(local_stream_socket &socket,
 /// A generic type representing a client connection to a server. This typename
 /// can be used to write messages synchronously to the server.
 class ServerConnection : public std::enable_shared_from_this<ServerConnection> {
+ private:
+  struct Tag {};
```

Comment:
this has no meaning right? you're just using it as a trick to make it impossible to call the public constructor because it requires a private struct type?

Reply:
Yes you're right, comment added.